extensions/wikia/ImageReview/ImageReviewImageStatus.php

```diff
@@ -108,3 +79,39 @@ function efImageReviewDisplayStatus( ImagePage $imagePage, &$html ) {
 
 	return true;
 }
+
+function fetchReviewHistory( $dbr, $cityId, $pageId ) {
+	// TODO: fetch history from image
+
+	$res = $dbr->select(
+		'image_review_stats',
+		'*',
+		[
+			'wiki_id' => $cityId,
+			'page_id' => $pageId,
+		]
+	);
+
+	$reveiws = [];
+	while ( $row = $dbr->fetchObject( $res ) ) {
+		$data = [];
+		$data[] = User::newFromId( $row->reviewer_id )->getName();
+		$data[] = wfMsg( 'imagereview-state-' . $row->review_state );
```

Comment:
Instead of legacy deprecated `wfMsg` function,  we should use `wfMessage` here with a correct output format (depending on whether we need escaping or wikitext parsing for this message).

Also, this kind of message concatenation makes it much more difficult to establish where a given message is used in the code. Please extract this logic to a helper function or list the possible values of the full message key in a doc block :)